ts/parts/Interaction.ts

```diff
@@ -753,12 +757,26 @@ extend(Chart.prototype, /** @lends Chart.prototype */ {
     pan: function (
         this: Highcharts.Chart,
         e: Highcharts.PointerEventObject,
-        panning: string
+        panning: Highcharts.PanningOptions|boolean
     ): void {
 
         var chart = this,
             hoverPoints = chart.hoverPoints,
-            doRedraw: boolean;
+            panningOptions: Highcharts.PanningOptions,
+            doRedraw: boolean,
+            type: string;
+
+        if (typeof panning === 'object') {
+            panningOptions = panning;
+        } else {
+            panningOptions = {
+                enabled: panning,
+                type: 'x'
+            };
+        }
+
+        (chart.options.chart as any).panning = panningOptions;
```

Comment:
I assume that casting to `any` is needed here because the property (second `chart` in `chart.options.chart`) is optional. 

Casting to `any` in new code is depreciated in favor of resolving the problem another way. In this case, you could add an `if` statement that will ensure the existance of `chart.options.chart`.